drivers/ethernet/eth_native_posix.c

```diff
@@ -155,6 +155,7 @@ static int read_data(struct eth_context *ctx, int fd)
 	struct net_buf *frag;
 	u32_t pkt_len;
 	int ret;
+	int count = 0;
```

Comment:
move it to line 153, we usually do so pre-assigned values are declared first, then the non-pre-assigned.

Reply:
done